exist-core/src/test/xquery/xquery3/serialize.xql

```diff
@@ -846,51 +846,3 @@ function ser:serialize-xml-134() {
     }
     return serialize((1 to 4)!text{.}, $params)
 };
-
```

Comment:
Could you please rebase on current develop so that we can see that these tests are preserved and also pass.

Reply:
I moved these 4 tests to fnSerializeNewline.xqm - I found breaking down into more smaller test scripts helpful when debugging.

Reply:
I missed that by looking through the changes. I would like to see such a refactoring mentioned in the commit message to be able to understand what and more importantly why that was done.

Reply:
Point taken. I've added a bit to the PR comment.